app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, first_file, second_file = command.split()
+    second_file_path = second_file[:second_file.rfind("/")]
```

Comment:
it's better to use `os.path.split()` for this purpose, for example windows use `\` instead of `/`

Reply:
what about it?)

Reply:
I rewrite this part of code
Check please